### ISSUE TITLE
Prints warning and then throws exeption

### ISSUE THREAD

**Issue by ilhanyumer** [PHPMailer/PHPMailer#982]
I have `$mail = new PHPMailer(true);` and I have `catch (phpmailerException $e)` and then `catch (Exception $e)`.

The problem is that it throws a warning before exception. Thus it makes my JSON useless.

```
<br />
<b>Warning</b>:  stream_socket_enable_crypto(): SSL operation failed with code 1. OpenSSL Error messages:
error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed in <b>/var/www/vhosts/example.com/subdomains/mobile/httpdocs/library/mailer/class.smtp.php</b> on line <b>369</b><br />
{"success":false,"thisis":"phpmailerException","msg":"<strong>SMTP Error: Could not connect to SMTP host.<\/strong><br \/>\n"}
```

I use PHP 5.6.29 and according to `VERSION` file I use PHPMailer 5.2.16 and according to `class.phpmailer.php` file I use PHPMailer 5.2.17.

I know I can use `SMTPOptions`, in fact I have
```
if($row->SMTP_OPTIONS)
{
	$mail->SMTPOptions = json_decode($row->SMTP_OPTIONS, true);
}
```
The script works correctly if I set SMTP_OPTINS in the database. However I want the script to throw only the exception and not to print warning. This way the user can be informed that there is misconfiguration.

**Comment by Synchro** [comment]
I've pushed a fix for this in c100b092dfdd0336d63efb6f3fe3502522260683 - errors encountered when enabling TLS are now trapped and reported like other problems. Can you confirm this works for you?

**Comment by ilhanyumer** [comment]
Thanks! Now it works as you've described.
